internal/runtime/helper.go

```diff
@@ -128,3 +130,47 @@ func constructTransactions(
 
 	return txs, nil
 }
+
+func calculateRuntimeCosts(
+	key crypto.PrivKey,
+	account std.Account,
+	transactions uint64,
+	chainID string,
+	getMsg msgFn,
+	estimateFn EstimateGasFn,
+) (std.Coin, error) {
+	fmt.Printf("\n⏳ Estimating Gas ⏳\n")
+
+	// Estimate the fee for the transaction batch
+	// passing in the maximum block gas, this is just a simulation
+	txFee := common.CalculateFeeInRatio(
+		types.MaxBlockMaxGas,
+		common.DefaultGasPrice,
```

Comment:
Also, we need to fetch this from the chain, now we have an endpoint

Reply:
Resolved in [14c58b1](https://github.com/gnolang/supernova/pull/89/commits/14c58b1f818933f15c8d32c82673468be43e6a5b)